app/(dashboard)/mailboxes/[mailbox_slug]/dashboard/customTimeRangePicker.tsx

```
@@ -2,61 +2,195 @@
+      const toM = Number(toMstr);
+      if ([fromH, fromM, toH, toM].some((n) => isNaN(n))) return false;
+      return fromH < toH || (fromH === toH && fromM < toM);
+    }
+    
+    return true;
+  };
 
   const handleSelect = () => {
-    if (!date) return;
+    if (!dateRange?.from || !dateRange?.to || !isValidTimeRange()) return;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)